.devcontainer/contributor/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Radius CLI - Contributor",
```

Comment:
Something I wonder - does this need to be specific to the CLI? It feels like this is useful for any kind of contribution.

Reply:
You are right. I will rephrase it to Radius - Contributor